services/motion/builtin/builtin_test.go

```diff
@@ -790,24 +790,6 @@ func TestCheckPlan(t *testing.T) {
 		test.That(t, err, test.ShouldNotBeNil)
 	})
 
-	t.Run("non nil error state - ensure success", func(t *testing.T) {
```

Comment:
This test is no longer particularly useful, as we can't guarantee that PTGs will take any particular path to reach the goal.

Reply:
Interesting. I'll explore that.